lib.go

```diff
@@ -232,12 +232,18 @@ func ColumnNumberToName(num int) (string, error) {
 	if num < MinColumns || num > MaxColumns {
 		return "", ErrColumnNumber
 	}
-	var col string
+	estimatedLength := 0
```

Comment:
Could you provides benchmark data for this performance improvements?

Reply:
I wrote a benchmark test for the old and new version of the function, and I also made some changes (replacing a slice of runes with a slice of bytes).

[bencgcmp_result.txt](https://github.com/user-attachments/files/16086822/bencgcmp_result.txt)